doc/manual/src/installation/upgrading.md

```diff
@@ -1,25 +1,37 @@
 # Upgrading Nix
 
-First find the name of the current channel:
+> **Note**
+>
+> These upgrade instructions apply for regular Linux distributions where Nix was installed following the [installation instructions in this manual](./index.md).
+
+First, find the name of the current [channel](@docroot@/command-ref/nix-channel) through which Nix is distributed:
 
 ```console
 $ nix-channel --list
 ```
 
-Will return:
+By default this should return an entry for Nixpkgs:
 
 ```console
-$ nixpkgs https://nixos.org/channels/nixpkgs-23.05
+nixpkgs https://nixos.org/channels/nixpkgs-23.05
 ```
-Remove old channel:
+
+Check which Nix version will be installed:
 
 ```console
-$ nix-channel --remove nixpkgs
+$ nix-shell -p nix -I nixpkgs=channel:nixpkgs-23.11 --run "nix --version"
```

Comment:
Wouldn't it be better to link to [search.nixos.org](https://search.nixos.org/packages)? seems like a much faster option, you could also directly link to [information about the package](https://search.nixos.org/packages?channel=23.11&show=nix&from=0&size=50&sort=relevance&type=packages&query=nix).

Reply:
I try to keep the Nix manual self-contained because most of the action happens somewhere else and at a different pace. One point of integration for documentation is nix.dev, where we point to search.nixos.org very early on.

All in all, as said in the other comment, I really don't like the vanilla upgrade story and given the lack of clearly superior approaches wouldn't want to spend too much energy on it right now.